data/curated/border-patrol/intro.md

```diff
@@ -0,0 +1,13 @@
+DESCRIPTION
+
```

Comment:
```suggestion
```
This is a placeholder for the place to add your description. I think I'll make a video for each of these sections one of these days, because it's difficult to express some of this without showing an example!